yeat/tests/test_config.py

```diff
@@ -9,46 +9,13 @@
 
 import json
 import pytest
-from unittest.mock import patch
 from yeat.config import AssemblyConfigError
-from yeat.config.assembly import Assembly
 from yeat.config.config import AssemblyConfig
-from yeat.config.sample import Sample
 from yeat.tests import data_file
 
 pytestmark = pytest.mark.short
 
 
-def test_invalid_assembly_algorithm():
```

Comment:
Moved tests to different modules: `test_sample.py` and `test_assembly.py` for organization. Keeping `test_config.py` to contain tests testing for the `AssemblyConfig` class object.